pipelines/skylab/multiome/multiome.wdl

```diff
@@ -0,0 +1,115 @@
+version 1.0
+
+import "../../../pipelines/skylab/multiome/atac.wdl" as atac
+import "../../../pipelines/skylab/optimus/Optimus.wdl" as optimus
+
+workflow Multiome {
+    String pipeline_version = "1.0.0"
+
+    input {
+        # Optimus Inputs
+        String counting_mode = "sn_rna"
```

Comment:
@nikellepetrillo I think the WDL might not match what's currently on develop in beta-pipelines. Aseel made changes to clean up the input names so there was no confusion about which fastqs were ATAC vs. gex. https://github.com/broadinstitute/warp/blob/develop/beta-pipelines/multiome/multiome.wdl

Reply:
thanks for catching that @ekiernan 👍  should be all updated now